python/ray/node.py

```diff
@@ -626,11 +626,8 @@ def start_dashboard(self, require_dashboard):
                 if we fail to start the dashboard. Otherwise it will print
                 a warning if we fail to start the dashboard.
         """
-        if "RAY_USE_NEW_DASHBOARD" in os.environ:
-            stdout_file, stderr_file = None, None
-        else:
-            stdout_file, stderr_file = self.get_log_file_handles(
-                "dashboard", unique=True)
+        stdout_file, stderr_file = self.get_log_file_handles(
+            "dashboard", unique=True)
```

Comment:
Dashboard use a file logger by default (the `dashboard.log`), it's not necessary to redirect the stdout and stderr.

Reply:
stdout redirection can neither rotate log files nor fragment log files. It is not good for long running cluster. Three log files may confuse users. (`dashboard.out`, `dashboard.err`, `dashboard.log`)

Reply:
Hmm having a separate .err is useful though. Is there any way we can have .out (rotated) and .err?

Reply:
Another way to have a seperate .err with rotate feature (I have not tested):
- stdout_file, stderr_file = None, None
- Two `RotatingFileHandler` instances,
  - One level is `WARNING`, filename is `dashboard.err`.
  - The other level is `INFO`, filename is `dashboard.out`,  `--logging-level` only affects this one.

Reply:
I see. I am okay with mixed logs because I agree we should make all logs rotational. @mfitton Can you just revert back to the original (@fyrestone)'s implementation?